destinyitemmanager.com/index.html

```diff
@@ -1,167 +1,177 @@
+          the beta</em></a><img class="browser" src="images/Browser.png" width="654px" alt="Browser running Destiny Item Manager">
     </div>
-    <section class="whats-dim">
-      <div class="row">
-        <div class="col-5 info">
-          <div class="text-wrap">
-            <h2>What's DIM?</h2>
-            <h3>Weapons</h3>
-            <p>Forgot to grab your favorite hand cannon? Just click and drag it to your active charcter.</p>
-            <h3>Gear</h3>
-            <p>Need those Bones of Eao for the jumping puzzle? Fire up DIM and grab them.</p>
-            <h3>Loadouts</h3>
-            <p>Create the perfect loadout for Raids, Strikes, Trails, and any other loadout that woud be helpful.</p>
-          </div>
+  </div>
+  <section class="whats-dim">
+    <div class="row">
+      <div class="col-5 info">
+        <div class="text-wrap">
+          <h2>What's DIM?</h2>
+          <h3>Weapons</h3>
+          <p>Forgot to grab your favorite hand cannon? Just click and drag it to your active charcter.</p>
+          <h3>Gear</h3>
+          <p>Need those STOMP-EE5 for the jumping puzzle? Fire up DIM and grab them.</p>
+          <h3>Loadouts</h3>
+          <p>Create the perfect loadout for Raids, Strikes, Gambit, and any other loadout that woud be helpful.</p>
         </div>
-        <div class="col-5"><img class="image-shadow" alt="Gear Square" src="images/GearSquare@2x.jpg" width="379px"></div>
       </div>
-    </section>
-    <section class="easy">
-      <div class="row">
-        <div class="col-5 image"><img class="image-shadow" alt="Inspect Weapons" src="images/InspectWeaponShot@2x.jpg" width="379px"></div>
-        <div class="col-5">
-          <div class="text-wrap">
-            <h2>It Really Is That Easy.</h2>
-            <h3>Drop Rating Percentage</h3>
-            <p>Going for a Tier 12 build? Use DIM to see how good your roll is on armor. (Not available for Destiny 2 yet)</p>
-            <h3>Weapon Reviews / Ratings</h3>
-            <p>Review and rate your favorite guns, and let others know what you think.</p>
-            <h3>Tagging</h3>
-            <p>Organize your stuff by what to keep, what's your favorite, what to infuse and what to get rid of.</p>
-          </div>
+      <div class="col-5"><img class="image-shadow" alt="Gear Square" src="images/GearSquare@2x.jpg" width="379px"></div>
+    </div>
+  </section>
+  <section class="easy">
+    <div class="row">
+      <div class="col-5 image"><img class="image-shadow" alt="Inspect Weapons" src="images/InspectWeaponShot@2x.jpg"
+          width="379px"></div>
+      <div class="col-5">
+        <div class="text-wrap">
+          <h2>It Really Is That Easy.</h2>
+          <h3>Drop Rating Percentage</h3>
+          <p>Going for a Tier 12 build? Use DIM to see how good your roll is on armor. (Coming soon for Destiny 2)</p>
```

Comment:
We should probably remove this.

Reply:
Yeah also wasn't sure here, I know its really close to being in beta so I changed the wording to coming soon.